src/schema/v1/artist/__tests__/index.test.js

```diff
@@ -27,14 +27,14 @@ describe("Artist type", () => {
   })
 
   it("returns null for an empty ID string", () => {
-    return runQuery(`{ artist(id: "") { id } }`, context).then(data => {
+    return runQuery(`{ artist(id: "") { id } }`, context).then((data) => {
```

Comment:
There seems to be a lot of these prettier changes? There's a bit of noise in the diff.

Reply:
Yeah, I had to update all of these files _slightly_ -- I'm unclear on whether these changes are desired or not (they seem like what we want?)

Reply:
It feels like a redundant parantheses to me? I'm just not sure if locally you might have settings that are different from others.